.github/workflows/buid_and_release.yml

```diff
@@ -48,4 +50,4 @@ jobs:
         prerelease: false
         files: |
           omim.ttl
-          omim.sssom.tsv
+#          omim.sssom.tsv
```

Comment:
Commented out?

Reply:
Yeah, I tagged that on you here: #92 
let me know what you think